src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -1192,6 +1192,37 @@ public static class ValidationRules
                     }
                 });
 
+        // Add this to rule set as a breaking change for a future release.
+        /// <summary>
+        /// Validates that if a complex-typed property has a type or base type that is the same as
+        /// the complex type, then the complex type is nullable.
+        /// </summary>
+        public static readonly ValidationRule<IEdmStructuralProperty> RecursiveComplexTypedPropertyMustBeOptional =

```

Comment:
```suggestion
        public static readonly ValidationRule<IEdmStructuralProperty> RecursiveComplexTypePropertyMustBeOptional =

```

Not sure if this was a typo or not

Reply:
I borrowed this wording from the attached issue -- although I suppose that could have been a typo